projects/media-viewer/src/lib/styles/media.scss

```diff
@@ -350,3 +358,15 @@
     display: none;
   }
 }
+
+@media all and (max-width: 1000px) {
+  .hiddenIfLargerThan1000px, .hiddenIfLargerThan1000px * {
```

Comment:
im not sure the * works on class names. I know it works on element tags, but im sure it doesnt for class name.

Reply:
will add div.